### DISCUSSION TITLE
modified Brier score，the smaller，the better？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#144] [Q&A]
modified Brier score，the smaller，the better？
modified Brier score value is small，close to 0，signifies a good classifier？
the value range of modified Brier score is 0~1？

**Comment by ishwaran** [comment]
It's calibrated so that 1=random guessing and 0=perfect.

See https://luminwin.github.io/randomForestSRC/articles/getstarted.html "Specifying the Type of Predicted Value, Variable Importance and Error Rate"
